content/docs/user-guide/editor/viewport/_index.md

```diff
@@ -1,40 +1,121 @@
+
+![Image of the translation manipulator with the linear, planar, and surface manipulator highlighted](/images/user-guide/editor/transform-manipulator.png)
+
+The translation manipulator is composed of three different types of manipulators. The red, green, and blue arrows are _linear manipulators_, they correspond to the X-axis, Y-axis, and Z-axis, respectively.  You can **left-click** and drag a linear manipulator to move an entity along a single axis at a time.  Manipulators turn yellow when you select them.
+
+Use _Planar manipulators_, represented by squares, to move an entity along two axes while restricting movement on the unselected axis. 
+
+The yellow sphere at the center of the translation manipulator is the _surface manipulator_.  You can move an entity along the surface of other meshes that are visible in the 3D Viewport.  If there are no surfaces that the entity would intersect with, the entity will move a fixed distance in front of the camera.
+
+![Image of translation manipulator with a ghost axis](/images/user-guide/editor/ghost-axis.png)
+
+The translation manipulators display a red, blue, and green _ghost axis_ that indicates the original position of the entity while a manipulator is in use.
+
+### Rotation manipulator
+
+![Image of the rotation manipulator](/images/user-guide/editor/rotation-manipulator.png)
+
+The rotation manipulator is composed of two types of manipulators. A set of red, green, and blue circles are _angular manipulators_ that rotate the entity on the X-axis, Y-axis, and Z-axis, respectively.  The larger white circle is the _camera space manipulator_, which rotates the entity on the Editor camera's forward axis. **Left-click** and drag a manipulator to rotate the entity on an axis.
+
+![Image of the rotation manipulator with a yellow sector that shows how much the entity has been rotated](/images/user-guide/editor/rotation-feedback.png)
+
+The rotation manipulators display a yellow sector that indicates the original orientation of the entity while a manipulator is in use.
+
+### Scale manipulator
+
+![Image of the scale manipulator](/images/user-guide/editor/scale-manipulator.png)
+
+The scale manipulator is represented by a set of white cubes on an 3D axis.  You can **left-click** and drag away from the manipulator's center to increase the uniform-scale of an entity.
+
+## Pivots
+
+![Side-by-side images of an entity showing the locations of it's default and center pivot](/images/user-guide/editor/pivots.png)
+
+Entities with a three-dimensional shape may have more than one pivot point.  In the viewport, the pivot point is indicated by a small yellow cube.  The default pivot point is located at the local original of the entity, often this is set when the 3D object is modelled in a DCC tool or imported into O3DE.  The second pivot point of an entity is located at the geometric center of the object's volume.  You can switch between the two pivot points by pressing **P** on the keyboard.
```

Comment:
update: local original -> local origin

update modelled in a DCC tool or imported into O3DE -> modelled in a DCC tool **and** imported into O3DE